primitives/message-dispatch/Cargo.toml

```diff
@@ -10,9 +10,20 @@ license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 bp-runtime = { path = "../runtime", default-features = false }
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false }
 
+# Substrate Dependencies
+
+frame-support = { git = "https://github.com/paritytech/substrate", branch = "master" , default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate", branch = "master" , default-features = false }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" , default-features = false }
```

Comment:
You don't need these two dependencies here

```suggestion
```

Reply:
done it